src/app/check-out/check-out.component.html

```diff
@@ -32,6 +32,9 @@ <h2  style="margin-left:60px ">Booking</h2>
           ngModel class="form-control" required #address="ngModel"></textarea>
           </div>
           <button class="btn btn-success " style="margin-left:15px "  (click)="placeOrder()">Book Now</button>
```

Comment:
please consider adding sty
```suggestion
          <button class="btn btn-success " style="margin-left:15px; margin-bottom:15px; "  (click)="placeOrder()">Book Now</button>
```